### PR TITLE
Proxy handling / additional send method

### PR DESCRIPTION
**Pull request by harald-mueller**
Note variable no_proxy for hosts that should not connect via the proxy.
Add method to send an iq frame that not contains a query element.

### CODE REVIEW

File: xmpp.go [hunk 1]
```diff
@@ -1,4 +1,7 @@
-// Copyright 2011 The Go Authors.  All rights reserved.
+// Copyright 2011 The Go Authors.
+// Copyright 2018 Siemens AG
```
Comment (mattn) [inline xmpp.go]: if you want to reserve this patch, I can't merge this.

Comment (harald-mueller) [reply xmpp.go]: i have reverted that, was a try to match the company rules. Hope that this is now ok.

**Comment by mattn** [comment]
merged. thank you

https://github.com/mattn/go-xmpp/commits/master

**Comment by mattn** [comment]
Thank you.

**Pull request merged**